### DISCUSSION TITLE
建议增加能够最小化至托盘的控制台运行模式

### DISCUSSION THREAD

**Discussion by crisbon** [PBH-BTN/PeerBanHelper#305] [讨论]
Windows 上目前只有四种运行模式：GUI，GUI Safe mode，GUI Silent，NoGUI。然而所谓的 GUI 仅仅是显示log而已，所以我希望能够使用控制台模式，同时将它最小化至托盘，但是目前NoGUI模式无法最小化至托盘，建议增加能够最小化至托盘在后台运行的控制台模式。具体效果参见Jackett。

**Comment by Ghost-chu** [comment]
使用 GUI 模式启动时添加 silent 附加参数即可在启动时直接静默启动到托盘。控制台模式由于其窗口并非由 PBH 管理，而是完全由系统管理，无法进行任何操作。

如果您完全不喜欢 GUI，也不想看到控制台窗口，请考虑使用安装程序安装为 Windows 系统服务（注意：安装为服务后，数据存储目录会被 Windows 保护，更改配置文件和访问数据均需要管理员权限）。
